core/client/api/content.js

```diff
@@ -7,6 +7,16 @@ const load = async (contentType, slug) => {
   return response.data;
 };
 
+const create = async (contentType, slug, data) => {
+  const response = await axios.post(`${HOST}/_api/content/${contentType}/${slug}/new`, data);
```

Comment:
you can omit `${slug}/new` and just do post request to server passing the slug in the body. Post to `_api/content/${contentType}` would be sufficient in REST to tell server to create something. you can read more here about the REST overview - https://medium.com/the-coding-matrix/rest-101-ea5e54e3290c